extensions/panache/hibernate-reactive-rest-data-panache/deployment/pom.xml

```
@@ -10,7 +10,7 @@
     </parent>
 
     <artifactId>quarkus-hibernate-reactive-rest-data-panache-deployment</artifactId>
-    <name>Quarkus - Hibernate Reactive Reactive data with Panache - Deployment</name>
+    <name>Quarkus - Hibernate Reactive data with Panache - Deployment</name>
```

```suggestion
    <name>Quarkus - Hibernate Reactive REST Data with Panache - Deployment</name>
```
And probably worth checking the parent and runtime modules are consistent with this naming.

---

Just checked the parent / runtime module and they are properly named. I also check the pattern `reactive[ -]data[^sb]` (`sb` to exclude datasource and database) and did not find anything except `REST Reactive data` (fixed).